src/sentry/static/sentry/app/views/settings/components/dataPrivacyRulesPanel/dataPrivacyRulesPanel.tsx

```diff
@@ -120,6 +135,38 @@ class DataPrivacyRulesPanel extends React.Component<Props, State> {
     }
   }
 
+  loadSelectorSuggestions = async () => {
+    const {organization, project} = this.context;
+
+    const queryParams = [
+      ['project', project?.id],
+      ['eventId', this.state.selectedEventId],
+    ]
+      .filter(([_k, v]) => !!v)
+      .map(([k, v]) => `${k}=${v}`)
+      .join('&');
+
+    const selectorSuggestions: Array<Suggestion> = [];
+
+    const rawSuggestions = await this.api.requestPromise(
+      `/organizations/${organization.slug}/data-scrubbing-selector-suggestions/?${queryParams}`,
+      {method: 'GET'}
```

Comment:
If you make your query parameters into an object you could pas the object into `requestPromise()` as the `query` option.

```typescript
this.api.requestPromise(
      `/organizations/${organization.slug}/data-scrubbing-selector-suggestions/?${queryParams}`,
      {method: 'GET', query: {eventId: this.state.selectedEventId, project: project?.id}}
);
```